api/v2/api.proto

```diff
@@ -197,5 +198,5 @@ service Dex {
   // VerifyPassword returns whether a password matches a hash for a specific email or not.
   rpc VerifyPassword(VerifyPasswordReq) returns (VerifyPasswordResp) {};
   // Sign Vehicle Token takes a valid JSON that includes the userID, Id for privileges requested and vehicleID and returns a token
-  rpc GetVehiclePrivilegeToken(GetVehiclePrivilegeTokenReq) returns (GetVehiclePrivilegeTokenResp) {};
+  rpc GetCustomToken(SignTokenRequest) returns (SignTokenResp) {};
```

Comment:
```suggestion
  rpc SignToken(SignTokenReq) returns (SignTokenResp) {};
```
Preserves the variable naming pattern and the verb "sign" means more to my ear. What do you think?